client/app/telecom/pack/slots/emailPro/pack-emailPro.html

```diff
@@ -0,0 +1,38 @@
+<div id="pack-hostedEmail"
+     data-ng-controller="PackEmailProCtrl as EmailPro">
+    <div class="oui-tile__body"
```

Comment:
Why you didn't use [`<oui-tile>`](http://master.ui-kit.ovh/#!/oui-angular/tile) here?